.github/workflows/binary-combine-ios.yml

```diff
@@ -0,0 +1,54 @@
+name: IOS Build Combine
+
+on:
+  workflow_call:
+   inputs:
+      os:
```

Comment:
Same here - this can only run on macos, so what's the point of specifying the os externally?